src/index.js

```diff
@@ -14,13 +14,13 @@ module.exports = class Layer {
 
     var child = this._render()
 
-    return React.render(child, this._mountPoint, cb);
+    return ReactDOM.render(child, this._mountPoint, cb);
```

Comment:
There is actually one more change that should be made here, use: `ReactDOM.unstable_renderSubtreeIntoContainer` instead of `render()` it has the same signature, but passes context along correctly.